File: 1512-number-of-good-pairs/1512-number-of-good-pairs.go
func numIdenticalPairs(nums []int) int {
    number_of_good_pairs := 0
    for i := 0 ; i <len(nums)-1 ; i++{
        for j := i+1 ; j <len(nums); j++{
            if nums[i] == nums[j]{
                number_of_good_pairs++
            }
        }
    }
    return number_of_good_pairs
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        cntr = 0 
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i] == nums[j] and i != j:
                    cntr += 1
        return cntr
        